sentry_sdk/decorators.py

```diff
@@ -0,0 +1,89 @@
+import logging
+from typing import Optional
+from functools import wraps
+import inspect
+
+import sentry_sdk
+
+logger = logging.getLogger(__name__)
+
+
+def sentry_traced(
+    func=None,
+    *,
+    transaction_name: Optional[str] = None,
+    op: Optional[str] = None,
+):
+    """
+    Function decorator to start a child under the existing current transaction or
+    under a new transaction, if it doesn't exist.
+
+    Args:
+        transaction_name: the name of the new transaction if no transaction already
+            exists. If a transaction is found in current scope, this name is ignored.
+            If no transaction is found and no transaction name is provided, no action
+            is taken.
+        op: the name of the child. Defaults to the decorated function name.
+
+    Returns:
+        a decorated function executing within a Sentry transaction child
+
+    Usage:
+    @sentry_traced
+    def my_function():
+        ...
+
+    @sentry_traced(transaction_name="new_tx")
+    async def my_async_function():
+        ...
+
+    @sentry_trace(op="child_name")
```

Comment:
Typo: missing `d` at the end of `sentry_trace`.

Reply:
Done